### PR TITLE
Remove some legacy code

### PR DESCRIPTION
**Pull request by mcm-odoo**
task 3439226

### CODE REVIEW

File: addons/website_payment/static/src/js/website_payment_form.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 /** @odoo-module **/
 
-import core, { _t } from "@web/legacy/js/services/core";
+import core from "@web/legacy/js/services/core";
```
Comment (chevalierv) [inline addons/website_payment/static/src/js/website_payment_form.js:3]: Why is `core` left in this file?

Comment (mcm-odoo) [reply addons/website_payment/static/src/js/website_payment_form.js:3]: This file still uses the bus exported by core (see line 15/16 in the `start` function)

**Comment by jpp-odoo** [comment]
@robodoo r+
@robodoo rebase-ff

**Comment by sdegueldre** [comment]
@robodoo r- to let the assets common PR go first

**Comment by mcm-odoo** [comment]
Will be done in https://github.com/odoo/odoo/pull/133153

**Pull request closed without merging**